### PR TITLE
fix: smartbridge validation in `SendTransfer`

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
 - [x] Add validation hook for `SendTransfer` form
 - [x] Fix `InputCount` to update max letter-count label when typing
 - [x] Add smartbridge validation in `SendTransfer`

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-7uvk5w Add warning / error to Smartbridge Input](https://app.clickup.com/t/7uvk5w)

File: src/domains/transaction/validations/SendTransfer.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	}),
+	senderAddress: () => ({
+		required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+			field: t("COMMON.SENDER_ADDRESS"),
+		}),
+	}),
+	smartbridge: () => ({
+		required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+			field: t("COMMON.SMARTBRIDGE"),
+		}),
```
Comment (dated) [inline src/domains/transaction/validations/SendTransfer.ts:15]:
```suggestion
```

**Review by dated — COMMENTED** [review]
can you also replace
https://github.com/ArkEcosystem/desktop-wallet/blob/ef057d6bcef478c7cb8870ad79240ff820f7cd78/src/domains/transaction/pages/SendTransfer/Step1.tsx#L47-L49
with (without the div!)
```jsx
<FormLabel label="Smartbridge" required={false} />
```

looks good otherwise 👍

File: src/domains/transaction/validations/SendTransfer.ts (hunk 1 above)
Comment (dated) [reply src/domains/transaction/validations/SendTransfer.ts:15]: The smartbridge is an optional field :) don't worry about that <div>, I'll fix it in a separate PR as there are more adjustments needed 👍

Comment (goga-m) [reply src/domains/transaction/validations/SendTransfer.ts:15]: Was confused due to the fact that I saw the required dot :-)

**Review by luciorubeens — APPROVED** [review]

**Pull request merged**